### ISSUE TITLE
SteamVR tracking not responding

### ISSUE THREAD

**Issue by PTVoltz** [HipsterSloth/PSMoveSteamVRBridge#77]
OK, so I've got SteamVRBridge, and have set it up to track a virtual HMD with a yellow ping-pong ball. So far, it seems to be working - the position is tracked fine in the config tool.
However, for whatever reason the position in SteamVR does not change. I've run the driver installer, it's showing the cameras within the VR space, but it isn't actually tracking anything.

I've tried various things (uninstalling/reinstalling, unlinking the driver and restarting, etc) but the only idea that I currently have is that there's a problem somewhere with one of my other drivers - namely, TrueOpenVR.

I'm using a DIY headset, made using TrueOpenVR on the FreeTrack setting (Along with an Arduino, a screen, and FreePIE) for the stereoscopic view and rotation tracking. I have a feeling that the driver changes made by TOVR are in some way interfering with PSMS, however I have no way of checking this.

SteamVRBridge version - 1.6.0 (Latest)
SteamVR version - latest
TrueOpenVR version - 0.7 Portable (Latest)
If there are any logs  should be sending you, I can't find them...

**Comment by Krish312** [comment]
Have you found a fix yet? Im having the same issue. Head tracking works fine in the config tool but in steamVR only my hands work. Im using riftcat

**Comment by PTVoltz** [comment]
Not quite. I did figure out though, that (at least with my setup) you could run both the SteamVR Bridge AND FreePIE Bridge at the same time. Since I was already using FreePIE to track rotation, I just added the Headset position to the script (tracking a virtual controller) and it worked great.
